src/store/transactions/withdraw/withdraw.thunks.ts

```diff
@@ -252,7 +252,7 @@ function withdraw(
 
 function handleTransactionSuccess(dispatch: AppDispatch, accountIndex: string) {
   dispatch(openSnackbar("Transaction submitted"));
-  dispatch(push(`/accounts/${accountIndex}`));
+  dispatch(replace(`/accounts/${accountIndex}`));
```

Comment:
Shouldn't we consider a more resilient solution? I mean, I can enter the URL manually. Probably not a priority, but it feels like we should keep track of the state of the transaction (or ask the API) and activate the button accordingly.

It feels wrong to interfere with the default behavior of the back button. I think that the back button should send the user back to the previous screen (as expected) and the Withdraw button should be disabled once it has been clicked, as we currently do with the Exit.